app/keepers/keepers.go

```diff
@@ -585,3 +589,21 @@ func initParamsKeeper(
 
 	return paramsKeeper
 }
+
+type DefaultFeemarketDenomResolver struct{}
```

Comment:
```suggestion
type DefaultFeeMarketDenomResolver struct{}
```

Reply:
just as a note, Idm Feemarket over FeeMarket but just keep it consistent

Reply:
ok